eclair-core/src/test/scala/fr/acinq/eclair/StartupSpec.scala

```diff
@@ -17,8 +17,8 @@
 package fr.acinq.eclair
 
 import com.typesafe.config.{Config, ConfigFactory}
-import fr.acinq.bitcoin.scalacompat.Crypto.PublicKey
-import fr.acinq.bitcoin.scalacompat.{Block, SatoshiLong}
+import fr.acinq.bitcoin.scalacompat.Crypto.{PrivateKey, PublicKey}
```

Comment:
Imports need to be cleaned up.

Reply:
done in https://github.com/ACINQ/eclair/pull/2565/commits/a3da92a3f0bac4f0d093ce5548eb89f5b1287ed1